Tests/DatadogTests/Datadog/Utils/SwiftUIExtensionsTests.swift

```diff
@@ -0,0 +1,52 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+#if canImport(SwiftUI)
+
+import XCTest
+import SwiftUI
+@testable import Datadog
+
+@available(iOS 13, *)
+class CustomHostingController: UIHostingController<AnyView> {}
+
+@available(iOS 13, *)
+final class TestView: View {
+    var body = EmptyView()
+}
+
+class SwiftUIExtensionsTests: XCTestCase {
+    func testSwiftUIViewTypeDescription() {
+        guard #available(iOS 13, *) else {
+            return
+        }
+
+        let view = TestView().trackRUMView(name: "test")
```

Comment:
This is test for agnostic `SwiftUI` extension, so IDK if we want to mix it with RUM specifics. If we want to test `typeDescription` after applying modifier, it seems better to use a default one (e.g. apply background color or sth). WDYT?

Reply:
Yes, makes sense 👍